packages/fxa-content-server/tests/functional/lib/helpers.js

```diff
@@ -467,6 +482,7 @@ const clearBrowserState = thenify(function(options) {
         return this.parent.then(clear123DoneState({ untrusted: true }));
       }
     })
+    .then(cleanMemory())
```

Comment:
And here is the dumb place I stuck a `cleanMemory` call - at the tail end of `clearBrowserState` called by most tests.